packages/techdocs-cli/src/commands/index.ts

```diff
@@ -70,6 +70,11 @@ export function registerCommands(program: Command) {
       'Attempt to ensure an index.md exists falling back to using <docs-dir>/README.md or README.md in case a default <docs-dir>/index.md is not provided.',
       false,
     )
+    .option(
+      '--defaultPlugin [defaultPlugins...]',
```

Comment:
Sorry, just thought about one more thing:

Any particular reason why this is singular? The description is plural so it's a bit confusing. If we support multiple, can we specify in the description how to provide multiple vs one?

Reply:
Yeah, thought about the same while implementing it. I used the singular term, as it can be multiple, but doesn't have to - like the `--config` of the `config:print`.

So I can define the default plugins by adding them one by one - or as one array`techdocs-cli generate`:

```sh
techdocs-cli generate --defaultPlugin section-index --defaultPlugin my-custom-plugin

# or
techdocs-cli generate --defaultPlugin section-index my-custom-plugin
```

See also: https://github.com/tj/commander.js#variadic-option

---

Short, I think singularity fits best here, but I am also fine with plural.

Reply:
Ah I see, sounds good!